kenjutsu/kenjutsu.py

```diff
@@ -114,13 +114,22 @@ def reformat_slice(a_slice, a_length=None):
                     stop_i = False
 
     # Catch some known empty slices.
-    if (step > 0) and (stop == 0):
+    if stop_i and (start == stop):
```

Comment:
The not `None` check here for `stop` is probably not necessary as `start` is always defined. Still it is good to keep this check in case things change above.